style.css

```diff
@@ -0,0 +1,267 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.container{
+    display: grid;
+    grid-template: 
+    "header header header" 35px
+    "left-bar main right-bar" 1fr /
+    250px 1fr 300px;
+}
+/* .container img{
+    object-fit: cover;
+} */
+
+       /* HEADER */
+.header{
+    grid-area: header;
+    display: grid;
+    grid-template-columns: 250px 1fr 300px;
+    grid-template-rows: 35px;
+    top: 0px;
+    position: absolute;
+    z-index: -1;
+    width: 100%;
+}
+.logo{
+    align-self: center;
+    margin-left: 60px;
+}
+.home{
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    border: 1px solid #eceef0;
+    padding: 9px;
+}
+.header h1{
+    font-size: 15px;
+}
+.sparkles{
+    justify-self: end;
+    margin-right: 30px;
+    margin-top: 4px;
+}
+.header div, .header .lence{
+    align-self: center;
+}
+.search{
+    display: grid;
+    grid-template-columns: 30px 1fr;
+    background-color: #eceef0;
+    margin-left: 10px;
```

Comment:
here, to keep you code concise, you could use a shorthand property to set padding and margin